src/components/pages/HomePage.js

```diff
@@ -1,41 +1,230 @@
-import { React,useState } from "react";
+import { React, useState } from "react";
+import { Link } from "react-router-dom";
 import Graph from "../reusable/Graph";
 import Navbar from "../reusable/NavBar";
 import Sidebar from "../reusable/SideBar";
 
 const HomePage = () => {
+  // Currently hardcoded but will eventually come from API
+  const [chartData, setChartData] = useState([
+    50, 41, 35, 51, 4, 62, 262, 91, 134,
+  ]);
 
-    // Currently hardcoded but will eventually come from API
-    const [chartData, setChartData] = useState([1000,50,1])
-    const [categories, setCategories] = useState([
-      "Jan",
-      "Feb",
-      "Mar",
-    ])
-  
-    /*
+  const [categories, setCategories] = useState([
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "Jul",
+    "Aug",
+    "Sep",
+  ]);
+
+  const [selectedTabFilter, setSelectedTabFilter] = useState("investor");
+  const [selectedGraphFilter, setSelectedGraphFilter] = useState("Today");
+
+  const graphFilters = {
+    DAY: "Today",
+    FIVE: "Past 5 days",
+    MONTH: "Past month",
+    YEAR: "Past Year",
+  };
+
+  const tabFilters = {
+    INVESTOR: "investor",
+    JOB: "job",
+    history: "history",
+  };
+
+  const [stockData, setStockData] = useState([
```

Comment:
For the home page there should not be stock data

Reply:
We will have to discuss this since there is stock data displayed on the Homepage that is not within the graph component.

Reply:
@ryPattillo